packages/cmf/src/middlewares/http/index.md

```diff
@@ -0,0 +1,121 @@
+# HTTP middleware
+
+The HTTP middleware is loaded by default. It manages the HTTP methods
+* OPTIONS
+* GET
+* HEAD
+* PATCH
+* POST
+* PUT
+* DELETE
+* TRACE
+* CONNECT
+
+It will intercept those actions, but **keep in mind that the easiest way to use it is through the `HTTP actions utility`** (see below) :
+```javascript
+const action = {
+    type: 'POST', // or one of the HTTP methods
+    body: {
+        .. // the optional body to send
+    },
+    transform(jsonResponse) {
+        ... // the optional transformation to perform on the response before dispatch
+    },
+    onSend: 'MY_REQUEST_SENT', // the optional action type to dispatch on fetch
+    onError: 'MY_REQUEST_ERROR', // the optional action or action creator to dispatch on fetch error
+    onResponse: 'MY_REQUEST_RESPONSE', // the optional action or action creator to dispatch on fetch complete
+}
+dispatch(action);
+```
+
+| Argument | Type | Description | Mandatory |
+|---|---|---|---|
+| type | string | One of the supported HTTP methods. | true |
+| body | object | The body to send. | false |
+| transform | function | The transformation method to call on response, to adapt it before dispatch. | false |
+| onSend | string | The action type to dispatch before fetch. This is optional but CMF will still dispatch an `@@HTTP/REQUEST` action on every request. By providing this option, CMF will dispatch an action for your specific case after the global one. | false |
+| onError | string &#124; function | The action or action creator to dispatch on fetch error. This is optional because all http error will trigger an `@@HTTP/ERRORS` action with error details. By providing this option, CMF dispatch an action for your specific case after the global one.| false |
```

Comment:
Is this still the case ? As we don't have `@@HTTP/ERRORS` when we give an onErrorHandler